### PR TITLE
Update setting-up-databricks.md

### PR DESCRIPTION
**Pull request by mirnawong1**
replace sql endpoint w sql warehouse and add callout

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md [hunk 1]
```diff
@@ -35,7 +39,7 @@ Before starting this tutorial, you will need the following:
 
 2. For the purpose of this tutorial, you will be selecting AWS as our cloud provider but if you use Azure or GCP internally, please choose one of them. The setup process will be similar.
 3. Check your email to complete the verification process.
-4. After setting up your password, you will be guided to choose a subscription plan. You will need to select either the `Premium` or `Enterprise` plan to access the SQL Compute functionality, required for using the SQL Endpoint for dbt. We have chosen `Premium` for this tutorial. Click `Continue` after selecting your plan.
+4. After setting up your password, you will be guided to choose a subscription plan. You will need to select either the `Premium` or `Enterprise` plan to access the SQL Compute functionality, required for using the SQL warehouse for dbt. We have chosen `Premium` for this tutorial. Click `Continue` after selecting your plan.
```
Comment (matthewshaver) [inline website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md]:
```suggestion
4. After setting up your password, you will be guided to choose a subscription plan. Select the `Premium` or `Enterprise` plan to access the SQL Compute functionality required for using the SQL warehouse for dbt. We have chosen `Premium` for this tutorial. Click `Continue` after selecting your plan.
```

File: website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md [hunk 2]
```diff
@@ -15,9 +15,13 @@ This project will walk you through:
 
 - Setting up a Databricks account
 - Loading training data into your Databricks account
-- Configuring a SQL endpoint in Databricks
+- Configuring a SQL warehouse (formerly SQL endpoint) in Databricks
 - Connecting dbt Cloud and Databricks
 
+:::info Note 📌 
+Databricks SQL Endpoints' name has changed and is now called [**SQL Warehouses**](https://docs.databricks.com/sql/admin/sql-endpoints.html).
```
Comment (matthewshaver) [inline website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md]: Looks like Databricks doesn't capitalize this so changing here for consistency with the rest of the doc

```suggestion
Databricks SQL Endpoints' name has changed and is now called [**SQL warehouses**](https://docs.databricks.com/sql/admin/sql-endpoints.html). 
```

File: website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md [hunk 3]
```diff
@@ -180,7 +184,7 @@ Congratulations! At this point, you have created a Databricks account, loaded tr
     </div>
 
 2. Select the dbt tile under `Data preparation and transformation`.
-3. Click on `Next` when prompted to `Connect to partner`. This action will create a service principal, PAT token for that service principle, and SQL Endpoint for the dbt Cloud account to use. This does mean that you will have two SQL Endpoints at your disposal from the previous step and from using Partner Connect.
+3. Click on `Next` when prompted to `Connect to partner`. This action will create a service principal, PAT token for that service principle, and SQL warehouse for the dbt Cloud account to use. This does mean that you will have two SQL Warehouses at your disposal from the previous step and from using Partner Connect.
```
Comment (matthewshaver) [inline website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md]:
```suggestion
3. Click on `Next` when prompted to `Connect to partner`. This action will create a service principal, PAT token for that service principle, and SQL warehouse for the dbt Cloud account to use. This does mean that you will have two SQL warehouses at your disposal from the previous step and from using Partner Connect.
```

**Review by runleonarun — APPROVED** [review]
This was reviewed and the changes made, so it look ready to 🚢

**Pull request merged**